LeaderboardBackend/Services/UserService.cs

```diff
@@ -1,4 +1,4 @@
-using System.IdentityModel.Tokens.Jwt;
+using System.IdentityModel.Tokens.Jwt;
```

Comment:
line ending change for reasons 🤷 
I guess I committed it with the wrong one before